src/Highlighter.test.js

```diff
@@ -372,4 +374,17 @@ describe('Highlighter', () => {
     expect(node.classList.contains('test-class')).to.equal(true)
     expect(node.dataset.testAttribute).to.equal('data attribute content')
   })
+
+  it('should use :unhighlightTag if provided', () => {
```

Comment:
Could you also add tests to prove that `highlightTag` can be a functional/class component?